docs/why.rst

```diff
@@ -57,81 +57,71 @@ Adding an attribute to a class concerns only those who actually care about that
 …namedtuples?
 -------------
 
-The difference between :func:`collections.namedtuple`\ s and classes decorated by ``attrs`` is that the latter are type-sensitive and require less typing as compared with regular classes:
+:func:`collections.namedtuple`\ s are tuples with names, not classes. [#history]_
+Since writing classes is tiresome in Python, now and then someone discovers all the typing they can save and get really excited.
+However that convenience comes at a price.
 
+The most obvious difference between ``namedtuple``\ s and ``attrs``-based classes is that the latter are type-sensitive:
 
 .. doctest::
 
    >>> import attr
-   >>> @attr.s
-   ... class C1(object):
-   ...     a = attr.ib()
-   ...     def print_a(self):
-   ...        print(self.a)
-   >>> @attr.s
-   ... class C2(object):
-   ...     a = attr.ib()
-   >>> c1 = C1(a=1)
-   >>> c2 = C2(a=1)
-   >>> c1.a == c2.a
+   >>> C1 = attr.make_class("C1", ["a"])
+   >>> C2 = attr.make_class("C2", ["a"])
+   >>> i1 = C1(1)
+   >>> i2 = C2(1)
+   >>> i1.a == i2.a
    True
-   >>> c1 == c2
+   >>> i1 == i2
    False
-   >>> c1.print_a()
-   1
 
 
-…while a namedtuple is *explicitly* intended to behave like a tuple:
+…while a namedtuple is *explicitly* intended to behave like a tuple which means the type of a tuple is ignored:
 
 
 .. doctest::
 
    >>> from collections import namedtuple
    >>> NT1 = namedtuple("NT1", "a")
    >>> NT2 = namedtuple("NT2", "b")
-   >>> t1 = NT1._make([1,])
-   >>> t2 = NT2._make([1,])
+   >>> t1 = NT1(1)
+   >>> t2 = NT2(1)
    >>> t1 == t2 == (1,)
    True
 
+Other often surprising behaviors include:
 
-This can easily lead to surprising and unintended behaviors.
+- ``namedtuple``\ s are iterable and indexable which is not what you'd expect from a class.
+- Iterability also implies it's easy to accidentally unpack a ``namedtuple`` which leads to hard-to-find bugs. [#iter]_
+- ``namedtuple`` adds its methods to your instances whether you want or not. [#pollution]_
+- ``namedtuple``\ s are *always* immutable.
+  Not only does that mean that you can't decide for yourself whether your instances should be immutable or not, it also means that if you want to influence your class' initialization (validation?  default values?), you have to implement :meth:`object.__new__` which is a particularly hacky and error-prone requirement for a very common problem. [#immutable]_
+- To attach methods to a ``namedtuple`` you have to subclass it.
+  Again: common problem, hacky solution.
 
-Opinions on object immutability vary.
-With ``attrs``, the choice is yours.
-Immutable classes are created by passing a ``frozen=True`` argument to the :func:`attr.s` decorator.
-By default, however, classes created by ``attrs`` are regular Python classes and therefore mutable:
-
-.. doctest::
+All these things make ``namedtuple``\ s particular poor choice for public APIs because all your objects are irrevocably tainted.
```

Comment:
> All these things make `namedtuple`s **a** ~particular~ **particularly** poor choice for public APIs**,** because all your objects are irrevocably tainted.